packages/costa/src/client/entry.ts

```diff
@@ -202,3 +202,7 @@ process.on('message', (msg): void => {
   const proto = PluginProtocol.parse(msg);
   handlers[proto.op](proto);
 });
+
+process.on('unhandledRejection', (reason) => {
+  throw reason;
```

Comment:
We need to add a comment to explain why do we handle this event and throw this error :)

Reply:
Updated.